libflux/src/scanner/scanner.rl

```diff
@@ -52,7 +66,9 @@
         regex_lit => { tok = REGEX; fbreak; };
 
         # We have to specify whitespace here so that leading whitespace doesn't cause a state transition.
-        whitespace+;
+        whitespace;
```

Comment:
Why have you removed the `+`?

Reply:
The whitespace token defined above already has a trailing `+`, so it's redundant here.